src/pfe/Dockerfile_x86_64

```diff
@@ -159,6 +159,9 @@ RUN chmod -R +rx /file-watcher/idc
 
 WORKDIR /codewind-workspace
 
+WORKDIR /extensions
+COPY /extensions /extensions
```

Comment:
This `COPY` is failing in the build.  I think source path needs to be `src/pfe/extensions`?